dist/gcc-compatible/configure

```diff
@@ -282,6 +296,20 @@ if detect_arm; then
   fi
 fi
 
+if detect_x86; then
+  echo "... detected x86 platform"
+  echo "... $build_target does not support 256-bit arithmetic"
+  echo "BLACKLIST += $(ls *CP256*.c *_256.c *_Vec256.c | xargs)" >> Makefile.config
+  echo "... $build_target does not support legacy vale stubs"
+  echo "BLACKLIST += evercrypt_vale_stubs.c" >> Makefile.config
+  echo "#define Lib_IntVector_Intrinsics_vec256 void *" >> config.h
+  echo "$CC $CROSS_CFLAGS cannot compile 128-bit vector arithmetic, disabling"
+  echo "BLACKLIST += $(ls *CP128*.c *_128.c *_Vec128.c | xargs)" >> Makefile.config
+  echo "#define Lib_IntVector_Intrinsics_vec128 void *" >> config.h
+  echo "disabling the use of explicit_bzero"
+  echo "#define LINUX_NO_EXPLICIT_BZERO" >> config.h
```

Comment:
I would suspect that, say, a recent 32-bit linux distribution still supports bzero by virtue of having a recent glibc. Can we remove this bit and let the test at line 346 kick in?

Reply:
Yeah I can do that.